app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+  <%= f.label :name %><br />
+  <%= f.text_field :name, autofocus: true, autocomplete: "name", required: true%>
+  </div>
+
```

Comment:
- [ OPTIONAL ] Great job on the views but I think it would be great if the user could be able to see buttons for `signing up` and `signing in` on the home page so please add them since when I run it I can see the list of users and a button for sign out but when I run I should see two buttons for sign up and sign in  🔥😄 so kindly fix the routes to make the main root since the home should contain that 🔥💯

This the first thing I see

![image](https://github.com/edgardsoza/blog-app-edgard/assets/58553711/7c7de356-f2e6-4ab4-b31e-0ab11075f1ca)